core/src/test/scala/org/broadinstitute/dsde/rawls/util/RawlsSentryEventFilterSpec.scala

```diff
@@ -20,6 +20,17 @@ class RawlsSentryEventFilterSpec extends AnyFreeSpec {
       }
     }
 
+    // https://broadworkbench.atlassian.net/browse/BW-1125
+    "should filter out 'pet service account not found' errors" in {
```

Comment:
So there is a [test to check the other message is being filtered out](https://github.com/broadinstitute/rawls/blob/9ec1589582550fcf352a2122c1addd69761f9f4a/core/src/test/scala/org/broadinstitute/dsde/rawls/util/RawlsSentryEventFilterSpec.scala#L16-L21) and [test to make sure other messages are not being filtered out](https://github.com/broadinstitute/rawls/blob/9ec1589582550fcf352a2122c1addd69761f9f4a/core/src/test/scala/org/broadinstitute/dsde/rawls/util/RawlsSentryEventFilterSpec.scala#L34-L39). Is this what you were thinking about? 

  >A test where an error is thrown to test the Try matching?
  
So a test where we would throw and error [here](https://github.com/broadinstitute/rawls/blob/9ec1589582550fcf352a2122c1addd69761f9f4a/core/src/test/scala/org/broadinstitute/dsde/rawls/util/RawlsSentryEventFilterSpec.scala#L30) instead of calling the function `shouldSend()`?